includes/admin/settings/Status.php

```diff
@@ -138,6 +141,8 @@ private static function addWebhookStatusRow(): void
         $webhookSubscriptionService = new WebhookSubscriptionService();
         $allWebhooksPresent         = true;
 
+        (new AccountSettingsWebhook())->register();
```

Comment:
Ik vind dit wel een beetje een rare plek om dit te doen, de bedoeling is dat dit gewoon kijkt naar de status van zaken. 

Kan dit niet ergens in het opstartproces van de plugin? Desnoods maak je ergens een globale functie om te kijken of alle webhooks geregistreerd zijn, maar je moet niet webhooks aanmaken en settings aanpassen in code die alleen een stukje interface zou moeten renderen. Zo is het onlogisch en zorgt het er ook voor dat het heel moeilijk wordt om terug te vinden waar dit precies gedaan wordt.